yt/visualization/plot_modifications.py

```diff
@@ -3270,29 +3270,18 @@ def __call__(self, plot):
             plot.data.axis, plot.data, self.field_y, bounds, (nx, ny)
         )
 
-        vectors = np.concatenate((pixX[..., np.newaxis], pixY[..., np.newaxis]), axis=2)
-
         if self.texture is None:
-            prng = np.random.RandomState(0x4D3D3D3)
-            self.texture = prng.random_sample((nx, ny))
-        elif self.texture.shape != (nx, ny):
```

Comment:
I removed this check because `rlic.convolve` already runs sanity checks over its inputs, including this one.